com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -391,7 +391,7 @@ public void Update(GlobalDynamicResolutionSettings settings, Action OnResolution
         {
             ProcessSettings(settings);
 
-            if (!m_Enabled && !s_ActiveInstanceDirty)
+            if (!m_Enabled || !s_ActiveInstanceDirty)
```

Comment:
I am not sure what this change is doing? 

So s_ActiveInstanceDirty is determining whether we changed anything (is set to false at the end of this), so essentially this is going to call the update every 2 frames instead of every frame which of course is not what we want to do. 
Am I missing something?

Reply:
This should have been an or conditional.
Basically dont run this function if we already called it in the frame. 
The reason is that we have 2 loops for cameras within the same frame. 
Each loop calls UpdateAndUseCamera. So a camera calls Update() twice. But we only need to flush all the settings once per frame.

ClearSelectedCamera gets called at the end of the frame and sets this dirty flag to true.

THe problem is that this was false, it should be true. This fixes some crazy artifacting when resizing Hardware resolution scaler.

Reply:
Gotcha! I missed the 

> ClearSelectedCamera gets called at the end of the frame and sets this dirty flag to true.

part. 

Makes all sense.